ewatercycle/models/lisflood.py

```diff
@@ -259,11 +261,11 @@ def parameters(self) -> Iterable[Tuple[str, Any]]:
         """List the parameters for this model."""
         assert self.parameter_set is not None
         assert self.forcing is not None
-        # TODO fix issue #60
+        # TODO fix issue #84
         parameters = [
-            ('IrrigationEfficiency', self._get_textvar_value('IrrigationEfficiency')),
+            ('IrrigationEfficiency', self._get_textvar_value('IrrigationEfficiency'), 0.75),
```

Comment:
So here we pretend there's a default, even if it's not set in the model config? And then the model itself will fail?

Reply:
Yes, it will fail when area.map or area.nc file does not exist

Reply:
Reverted having a default in the model, will raise KeyError when MaskMap or IrrigationEfficiency can not be found in config.
An bmi model will return error when file from config does not exist